sycl/include/sycl/detail/spirv.hpp

```diff
@@ -176,7 +176,8 @@ EnableIfBitcastBroadcast<T, IdT> GroupBroadcast(T x, IdT local_id) {
 }
 template <typename Group, typename T, typename IdT>
 EnableIfGenericBroadcast<T, IdT> GroupBroadcast(T x, IdT local_id) {
-  T Result;
+  // assign x to support type T without default constructor
```

Comment:
```suggestion
  // Initialize with x to support type T without default constructor.
```

Reply:
Applied in https://github.com/intel/llvm/pull/7587/commits/f5881f95eab08e064f48332d4c54acd90a1004c9